### ISSUE TITLE
Anonymisation

### ISSUE THREAD

**Issue by HumanG33k** [lpereira/hardinfo#567]
It can be great to have an option to make anonymous report in order to join on report bug for ie.
I use the debian version of hardinfo.

i think information should not be share are : 
- network data
- hardware uuid 
- not service/usual software users.

**Comment by lpereira** [comment]
Good idea.  It's possible to not include a whole category of things in the report, but finer control to scrub potentially-sensitive data in reports isn't that easy to do.  There is some effort to refactor how some things are performed in HardInfo and it should be possible to add a flag in the future, but we're not there yet.

**Comment by HumanG33k** [comment]
also if a simple command line permit to use this kind of functionality it (hardinfo) can be use by other tools to make bug report. 

I will try to make a list of what data should not be share.
